pandas/tests/reshape/test_concat.py

```diff
@@ -2730,3 +2730,25 @@ def test_concat_datetimeindex_freq():
     expected = pd.DataFrame(data[50:] + data[:50], index=dr[50:].append(dr[:50]))
     expected.index._data.freq = None
     tm.assert_frame_equal(result, expected)
+
+
+def test_concat_copy_false_extension_array():
```

Comment:
I think this should be add to the class at `pandas.tests.extension.base.reshaping.BaseReshapingTests`.

It'll need to be restructured a bit to not refer to decimal specifically. It needs to work with the data provided by the fixture. See the other concat there for examples.

Reply:
SGTM. Moved the test to ` pandas.tests.extension.base.reshaping.BaseReshapingTests` and updated the test to use the data provided by the fixture. I also simplified the test to make it more clear.